### ISSUE TITLE
Crash Report 7.1: can't use vector drawables in notifications

### ISSUE THREAD

**Issue by mzorz** [wordpress-mobile/WordPress-Android#5635]
```
Fatal Exception: android.app.RemoteServiceException: Bad notification posted from package org.wordpress.android: Couldn't create icon: StatusBarIcon(pkg=org.wordpress.androiduser=0 id=0x7f020195 level=0 visible=true num=0 )
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1461)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5590)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1268)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1084)
       at dalvik.system.NativeStart.main(NativeStart.java)
``` 

We know this from before, i.e. look at this: #5497 and this https://github.com/wordpress-mobile/WordPress-Android/issues/4968#issuecomment-269999631

**Comment by tonyr59h** [comment]
Addressed in #5639, closing.

**Comment by maxme** [comment]
Reopening this one:

```
Fatal Exception: android.app.RemoteServiceException: Bad notification posted from package org.wordpress.android: Couldn't expand RemoteViews for: StatusBarNotification(pkg=org.wordpress.android user=UserHandle{0} id=30000 tag=null score=0: Notification(pri=0 icon=7f020196 contentView=org.wordpress.android/0x109007f vibrate=null sound=null defaults=0x0 flags=0x10 when=1492128509000 ledARGB=0x0 contentIntent=Y deleteIntent=Y contentTitle=11 contentText=158 tickerText=158 kind=[null] 2 actions))
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1484)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:146)
       at android.app.ActivityThread.main(ActivityThread.java:5602)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1283)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1099)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by tonyr59h** [comment]
There's a subtle difference in these crashes, `Couldn't create icon` vs `Couldn't expand RemoteViews for`.

Seems the second crash has to do with using unsupported views in a `RemoteView`, per [this StackOverflow question.](http://stackoverflow.com/questions/18558066/couldnt-expand-remoteviews)

**Comment by mzorz** [comment]
hah! we're just discussing this in DM @tonyr59h - awesome timing

**Comment by mzorz** [comment]
opened this new issue in #5648, closing this particular one as the icon cause is addressed in #5639

**Comment by hypest** [comment]
There are some crash reports happening on 7.1(386) so, reopening this.

**Comment by mzorz** [comment]
Was able to find the culprit after testing on Samsung Remote test labs - See [this comment](https://github.com/wordpress-mobile/WordPress-Android/issues/5648#issuecomment-294508145)

`ERROR|04-17 17:06:50.784|709|709||StatusBar|android.content.res.Resources$NotFoundException: File res/drawable/ic_reply_grey_32dp.xml from drawable resource ID #0x7f0201b9
`

Going to write the patch against this one

**Comment by daniloercoli** [comment]
Fixed in #5863
